templates/search/search_clients_daily_v8/query.sql

```diff
@@ -1,3 +1,16 @@
+-- Return the version of the search addon if it exists, null otherwise
+CREATE TEMP FUNCTION
+  get_search_addon_version(addon_list ANY TYPE) AS ((
+    SELECT
+      element.version
+    FROM
+      UNNEST(addon_list)
+    WHERE
+      element.addon_id = 'followonsearch@mozilla.com'
+    LIMIT
```

Comment:
upon further consideration, this should definitely be selecting mode_last:
```suggestion
    GROUP BY
      element.version
    ORDER BY
      COUNT(element.version) DESC,
      MAX(OFFSET) DESC
    LIMIT
```